nsxiv.1

```diff
@@ -366,7 +366,7 @@ The following mouse mappings are available in thumbnail mode:
 Select the image. Goes into image mode if double\-clicked.
 .TP
 .B Button3
-Mark/unmark the image. Dragging while holding Button3 down will mark/unmark
+Mark/unmark the image. Dragging while pressing Button3 down will mark/unmark
```

Comment:
You need to *hold* the button down while dragging for this to work. So I think the previous description here was better.

But if we want to use the word `pressing` then I think it should be `pressing down Button3` rather than `pressing Button3 down`

Reply:
In my opinion `holding down` would be the best.